apps/src/lab2/views/components/layout/layout.module.scss

```diff
@@ -63,7 +63,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-  height: 40px;
+  height: 2.5rem;
```

Comment:
Nit: it seems like styling for the `weblab2` customized workspace header doesn't seem general enough to be in a `lab2` layout stylesheet.
I had to kinda walk through which header container this was referring to. But it does seem like we want to keep the `useHorizontalLayout` and `useVerticalLayout` hooks in `lab2`. And having two different stylesheets within a component is kinda annoying...just thinking out loud 😁

Reply:
I was about to put more weblab2 styling in this lab2 file, so I ended up moving these styles into their own file in this PR: https://github.com/code-dot-org/code-dot-org/pull/68167